polaris-react/src/components/SkeletonThumbnail/SkeletonThumbnail.scss

```diff
@@ -1,22 +1,26 @@
 @import '../../styles/common';
 
-@mixin skeleton-thumbnail-size($size) {
-  height: thumbnail-size($size);
-  width: thumbnail-size($size);
-}
-
 .SkeletonThumbnail {
-  @include skeleton-content;
+  display: flex;
+  background-color: var(--p-surface-neutral);
+  border-radius: var(--p-border-radius-base);
+
+  @media screen and (-ms-high-contrast: active) {
+    background-color: grayText;
+  }
 }
 
 .sizeSmall {
-  @include skeleton-thumbnail-size(small);
+  height: 40px;
```

Comment:
The thing that concerns me about hard coding these is we lose the link between thumbnail component and that the values need to match. Maybe a shared custom property?

Reply:
Yes. You are totally right. I was thinking of creating a `ThumnailContainer` component but it just didn't seem necessary. I know the coverage tool will pick up on these values. My only concern is losing the connection between `SkeletonThumbnail` and `Thumbnail`.